### PR TITLE
gmrender-resurrect: cc96ede -> 0.0.8

### PR DESCRIPTION
**Pull request by ashkitten**
###### Motivation for this change

was marked broken, now updated with https://github.com/hzeller/gmrender-resurrect/pull/190 and tested to work. also, added myself as a maintainer because i plan to keep this up to date in the future as i use it

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @k0ral

### CODE REVIEW

**Comment by ashkitten** [comment]
the first release of gmrender-resurrect was just published, so i changed the pull request to reflect that. theoretically, we should be able to follow releases now!

File: pkgs/tools/networking/gmrender-resurrect/default.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
-{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, makeWrapper
+{ stdenv, fetchFromGitHub, fetchpatch, autoconf, automake, pkgconfig, makeWrapper
```
Comment (c0bw3b) [inline pkgs/tools/networking/gmrender-resurrect/default.nix]:
```suggestion
{ stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, pkgconfig, makeWrapper
```

**Review by c0bw3b — CHANGES_REQUESTED** [review]
You can replace autoconf+automake with `autoreconfHook` in `nativeBuildInputs` too.

Then the explicit definitions of `preConfigurePhases` and `autoconfPhase` aren't needed anymore.

**Comment by ashkitten** [comment]
just incorporated suggestions

**Review by c0bw3b — COMMENTED** [review]
Woah the diff is weird now! ~~Maybe you used another editor and it turned spaces into tabs.~~
\+ ~~inconsistent indentation in your let .. in block~~
\+ why scope fetchpatch? doesn't seem to be used

**Comment by ashkitten** [comment]
the let..in block was just one line before, so by adding more things i had to expand it out, thus the massive indentation change that made the diff effectively useless. sorry about that. i don't see any inconsistent indentation, though.

it seems i forgot to remove fetchpatch from before 0.0.8 was released and i was pulling in a patch to get it to compile. i'll remove that.

**Comment by c0bw3b** [comment]
> the let..in block was just one line before, so by adding more things i had to expand it out, thus the massive indentation change that made the diff effectively useless. sorry about that. i don't see any inconsistent indentation, though.

Oh ok! No you did nothing wrong then.
GH UI misled my eyes with those bold color blocks.. >;<

**Comment by ashkitten** [comment]
apparently there is an option to hide whitespace changes behind the cog icon in the diff ui
image

File: pkgs/tools/networking/gmrender-resurrect/default.nix [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-    platforms = platforms.linux;
-    broken = true;
-    maintainers = [ maintainers.koral ];
-  };
-}
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, makeWrapper, gstreamer
+, gst-plugins-base, gst-plugins-good, gst-plugins-bad, gst-plugins-ugly, gst-libav, libupnp }:
+
+let
+  version = "v0.0.8";
```
Comment (c0bw3b) [inline pkgs/tools/networking/gmrender-resurrect/default.nix]:
```suggestion
  version = "0.0.8";
```
Version [must start with a digit](https://nixos.org/nixpkgs/manual/#sec-package-naming) for Nix to parse it correctly

File: pkgs/tools/networking/gmrender-resurrect/default.nix [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
+  pluginPath = makePluginPath [ gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav ];
+in
+  stdenv.mkDerivation {
+    pname = "gmrender-resurrect";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "hzeller";
+      repo = "gmrender-resurrect";
+      rev = version;
```
Comment (c0bw3b) [inline pkgs/tools/networking/gmrender-resurrect/default.nix]: then
```suggestion
      rev = "v${version}";
```

**Review by c0bw3b — APPROVED** [review]
Builds and runs

**Pull request merged**